operators-manual/companion-web.md

```diff
@@ -103,6 +103,30 @@ To test the connection on a linux machine, bind to the port with "netcat" on the
 
 > **Note:** Topside as UDP Server mode is not recommended for master-slave setups. For the connection to be properly stabilished, the serial device must send some data. Only then the connection is stabilished and the topside is able to send data to the companion.
 
+This python script exemplifies how to run an UDP server with Python:
```

Comment:
weird english thing, this is one of those cases where you use 'a' in place of 'an' because it sounds better off the tounge, and that supersedes the normal vowel rule here.